src/components/Header.tsx

```diff
@@ -0,0 +1,24 @@
+import Icon from './Icon';
```

Comment:
Comments from previous components

Reply:
I think I fixed it on the last review-comment,
but Wi wonder if its more that just an example.